app/src/main/java/org/oppia/app/settings/profile/ProfileListActivity.kt

```diff
@@ -8,7 +8,8 @@ import javax.inject.Inject
 
 /** Activity that allows users to select a profile to edit from settings. */
 class ProfileListActivity : InjectableAppCompatActivity() {
-  @Inject lateinit var profileListActivityPresenter: ProfileListActivityPresenter
+  @Inject
+  lateinit var profileListActivityPresenter: ProfileListActivityPresenter
```

Comment:
Ditto here and elsewhere.

Reply:
Ktlint is a bit different, it says to be on the next line. Also, Ben discuss the same in yesterday's meeting